### ISSUE TITLE
`instanceof AjaxError` no longer possible since 7.0.0-rc.0

### ISSUE THREAD

**Issue by OliverJAsh** [ReactiveX/rxjs#6269]
## Bug Report

**Current Behavior**
In 7.0.0-beta.15 this did not error.

In 7.0.0-rc.0 and above, this results in an error.

```ts
import { AjaxError } from 'rxjs/ajax';
declare const error: unknown;
// Error:
// The right-hand side of an 'instanceof' expression must be of type 'any' or of a type assignable to the 'Function' interface type.
error instanceof AjaxError;
```

**Expected behavior**
No error

---

I glanced at the changelog but nothing stood out: https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md#700-rc0-2021-04-19.

**Comment by voliva** [comment]
I've found the source of this issue: AjaxErrorCtor signature is marked as @internal, and now the build step removes it.

I'll raise a PR now with a posible fix

**Comment by cartant** [comment]
Closed by #6275

**Comment by al-mart** [comment]
Isn't this a possible solution for both problems ? 
`class Forgeable {
  static isInstanceFlag = Symbol("isInstanceFlag");

  static [Symbol.hasInstance](obj) {
    return Forgeable.isInstanceFlag in obj;
  }
}`
